File: src/main/telemetry/api/Docs.java
package main.telemetry.api;

import java.io.File;
import java.io.IOException;
import java.io.BufferedWriter;
import java.io.FileWriter;

import java.net.URL;

import main.global.Items;
import main.telemetry.API;

public class Docs implements API {

  private final String docs = "> Properties Documentation <\n\nProperty Name | Description\ngui.defaultTheme | Changes the LAF of the MusicPlayer\nAvaliable Confs: regulardark, materia, onedark, arcdark, dracula, nord, gruvbox, vuesion, regularlight, solarized\n\nexplorer.defaultDir | Default Spawn Directory for the File Explorer:\n\".\", \"~\", \"/\"\n\nrunner.disableCache | Disable Caching\ntrue, false\n\ngui.defaultBoxSize | Change Button Spacing\n1 to 64";

  @Override
  public String run() throws IOException {
    if(!new File(Items.items[4]).exists())
      new File(Items.items[4]).createNewFile();
    BufferedWriter bw = new BufferedWriter(new FileWriter(returnLink().getFile()));
    bw.write(docs);
    bw.close();
    return docs;
  }

  @Override
  public URL returnLink() throws IOException {
    return new URL(Items.items[4]);
  }
  
}

File: kt/app/telemetry/api/Docs.kt
package app.telemetry.api

import app.telemetry.API
import kotlin.Throws
import java.io.IOException
import java.io.BufferedWriter
import java.io.FileWriter
import java.lang.StringBuilder
import app.telemetry.FileIntegrity
import app.CLI
import app.global.Items
import app.global.cli.CliType
import java.io.File
import java.net.URL

class Docs : API {
    private val docs = "> Properties Documentation <\n\nProperty Name | Description\ngui.defaultTheme | Changes the LAF of the MusicPlayer\nAvaliable Confs: regulardark, materia, onedark, arcdark, dracula, nord, gruvbox, vuesion, regularlight, solarized\n\nexplorer.defaultDir | Default Spawn Directory for the File Explorer:\n\".\", \"~\", \"/\"\n\nrunner.disableCache | Disable Caching\ntrue, false\n\ngui.defaultBoxSize | Change Button Spacing\n1 to 64"
    @Throws(IOException::class)
    override fun run(): String {
        if (!File(Items.items[4]).exists()) File(Items.items[4]).createNewFile()
        val bw = BufferedWriter(FileWriter(returnLink().file))
        bw.write(docs)
        bw.close()
        return docs
    }

    @Throws(IOException::class)
    override fun returnLink(): URL {
        return URL(Items.items[4])
    }
}